cmd/managementd/main.go

```diff
@@ -55,12 +55,14 @@ func main() {
 	router.HandleFunc("/camera-positioning", managementinterface.CameraPositioningHandler).Methods("GET")
```

Comment:
Let's call this `/camera` and `managementinterface.CameraHandler`. This may require some file renames too.